example.js

```diff
@@ -3,6 +3,9 @@ const { Client, Location, Poll, List, Buttons, LocalAuth } = require('./index');
 const client = new Client({
     authStrategy: new LocalAuth(),
     // proxyAuthentication: { username: 'username', password: 'password' },
+    /** Check https://github.com/pedroslopez/whatsapp-web.js/pull/3325 for full explanation. */
```

Comment:
The explanation and details should be either in example itself or in readme.md.
Imo, it's not a good practice to have a link to VCS (especially in github). 

We've seen many github repos got removed and we'll lose important notes :)